decidim-elections/app/views/decidim/elections/elections/_elections.html.erb

```diff
@@ -1,4 +1,23 @@
-<div class="row small-up-1 medium-up-2 large-up-3 card-grid">
+<% if @no_scheduled_elections %>
+  <div class="callout warning">
+    <%= t("decidim.elections.warnings.no_scheduled_elections_warning") %>
+  </div>
+<% elsif elections.length == 0 %>
```

Comment:
If you are checking that the _collection_ is empty (and not an Integer that equals zero), I would use `.zero?` as is more readable. ☺️

```suggestion
<% elsif elections.zero? %>
```